lib/proxy/db.c

```diff
@@ -64,12 +64,12 @@ static int db_busy(void *user_data, int busy_count) {
   /* If we're busy, then sleep for a short while, on the assumption that the
    * other process will finish its business with our tables.
    */
-  (void) pr_timer_usleep(PROXY_DB_SQLITE_MAX_RETRY_DELAY_MS);
+  (void) pr_timer_usleep(PROXY_DB_SQLITE_MAX_RETRY_DELAY_MS * 1000);
```

Comment:
This is actually fixing a bug; we were not _actually_ delaying long enough between retries.